axis.go

```diff
@@ -248,9 +248,9 @@ func (a *horizontalAxis) GlyphBoxes(*Plot) (boxes []GlyphBox) {
 		w := a.Tick.Label.Width(t.Label)
 		box := GlyphBox{
 			X: a.Norm(t.Value),
-			Rectangle: draw.Rectangle{
-				Min: draw.Point{X: -w / 2},
-				Max: draw.Point{X: w / 2},
+			Rectangle: vg.Rectangle{
```

Comment:
Well, it only breaks them twice in a row if they happen to `go get -u` between the time of the two PRs. In the future, maybe we should set a go1 tag on the gitrepo.